pkg/sqlqueries/SQL/mto/category/delete.sql

```diff
@@ -0,0 +1,12 @@
+DELETE FROM mto_category
```

Comment:
Are you planning on making a cascade statement so subcategories automatically delete when their parent delete?